tests/helpers/MockToolRegistry.ts

```
@@ -229,23 +248,27 @@ export class MockToolRegistry {
             };
             return { content: [{ type: 'text', text: JSON.stringify(fullStats) }] };
           
-          case 'agent_communication/clear_room_messages':
+          case 'agent_communication_clear_room_messages':
+            if (!args.confirm || args.confirm !== true) {
+              throw new ValidationError('Confirmation required: confirm must be set to true');
+            }
             if (!this.dataLayer.roomExists(args.roomName)) {
               throw new RoomNotFoundError(args.roomName);
             }
             const clearedCount = this.dataLayer.clearMessages(args.roomName);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)